frontend/app/routes/register.tsx

```diff
@@ -0,0 +1,115 @@
+import {
+  Alert,
+  Button,
+  TextField,
+  CircularProgress,
+  Typography,
+} from "@mui/material";
+import { Box } from "@mui/system";
+import type { ActionFunction, MetaFunction } from "@remix-run/node";
+import { json, redirect } from "@remix-run/node";
+import { Form, useActionData, useTransition } from "@remix-run/react";
+import { createApiClient } from "~/createApiClient";
+
+export const action: ActionFunction = async ({ request }) => {
+  const api = createApiClient();
+  const formData = await request.formData();
+  try {
+    await api.auth.registerRegisterApiAuthRegisterPost({
+      email: formData.get("email") as string,
+      password: formData.get("password") as string,
+    });
+    return redirect("/login", {
+      headers: {
+        "Set-Cookie": `isFirstTime=${true}`,
+      },
+    });
+  } catch {
+    return json({ msg: "Wrong/empty email or password. Or this user already exist" });
+  }
+};
+
+export const meta: MetaFunction = () => {
+  return { title: "Hello to Kronikarz" };
+};
+
+export default function Login() {
+  const transition = useTransition();
+  const actionMsg = useActionData();
+  return (
+    <>
+      <Box
+        sx={{
+          display: "flex",
+          justifyContent: "center",
+          alignItems: "center",
+          height: "100vh",
+          backgroundImage: "url(/background.png)",
+          backgroundRepeat: "no-repeat",
+          backgroundPosition: "center",
+          backgroundSize: "contain",
+          backgroundColor: "#e0edf2",
```

Comment:
```suggestion
```

I have set the backgroud color globally via mui theme. No need for magic colors now